### ISSUE TITLE
Set option defaults to those specified in Django admin

### ISSUE THREAD

**Issue by openjck** [mozilla/distribution-viewer#95]
In an email, Brendan mentioned that he would be "providing a csv... that includes default values for options."

We'll need to expose those defaults through the API (cc @robhudson) and set them accordingly on the front-end.

**Comment by openjck** [comment]
To clarify my earlier comment, when Brendan said he would provide default values for options, he meant that he would specify which charts should be linear by default, which charts should be log by default, which charts should show outliers by default, etc. These choices would override the global defaults.

I'm tempted to say we can take this out of V1, at least until we hear from users that the global defaults are problematic.

@robhudson: Where do you think Brendan should specify these per-chart defaults? He mentioned a CSV, but I know we added admin controls since then. Is this something you would see him doing through Django admin?

**Comment by robhudson** [comment]
Yes, those types of things I was thinking we would do through the Django admin and expose in the API. The front-end grabs all metrics and it can grab this metadata with the metrics also.
